spec/v3/services/settings_spec.rb

```diff
@@ -105,8 +105,9 @@
           'builds_only_with_travis_yml' => false,
           'build_pushes' => false,
           'build_pull_requests' => true,
-          'maximum_number_of_builds' => 20
+          'maximum_number_of_builds' => 4
         )
+        expect(repo.reload.user_settings.maximum_number_of_builds).to eq(4)
```

Comment:
This works locally w/ curl (it updates the database!) however it's not working in test yet.